flink/src/main/java/org/apache/iceberg/flink/data/vectorized/ConstantColumnVectors.java

```diff
@@ -0,0 +1,217 @@
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.flink.data.vectorized;
+
+import org.apache.flink.table.data.DecimalData;
+import org.apache.flink.table.data.TimestampData;
+import org.apache.flink.table.data.binary.BinaryStringData;
+import org.apache.flink.table.data.vector.BooleanColumnVector;
+import org.apache.flink.table.data.vector.BytesColumnVector;
+import org.apache.flink.table.data.vector.ColumnVector;
+import org.apache.flink.table.data.vector.DecimalColumnVector;
+import org.apache.flink.table.data.vector.DoubleColumnVector;
+import org.apache.flink.table.data.vector.FloatColumnVector;
+import org.apache.flink.table.data.vector.IntColumnVector;
+import org.apache.flink.table.data.vector.LongColumnVector;
+import org.apache.flink.table.data.vector.TimestampColumnVector;
+
+class ConstantColumnVectors {
+  private ConstantColumnVectors() {
+  }
+
+  static ColumnVector ints(Object constant) {
+    return new ConstantIntColumnVector(constant);
+  }
+
+  static ColumnVector longs(Object constant) {
+    return new ConstantLongColumnVongector(constant);
+  }
+
+  static ColumnVector booleans(Object constant) {
+    return new ConstantBooleanColumnVector(constant);
+  }
+
+  static ColumnVector doubles(Object constant) {
+    return new ConstantDoubleColumnVector(constant);
+  }
+
+  static ColumnVector floats(Object constant) {
+    return new ConstantFloatColumnVector(constant);
+  }
+
+  static ColumnVector decimals(Object constant) {
+    return new ConstantDecimalColumnVector(constant);
+  }
+
+  static ColumnVector timestamps(Object constant) {
+    return new ConstantTimestampColumnVector(constant);
+  }
+
+  static ColumnVector bytes(Object constant) {
+    return new ConstantBytesColumnVector(constant);
+  }
+
+  private static class ConstantIntColumnVector implements IntColumnVector {
+
+    private final Object constant;
+
+    private ConstantIntColumnVector(Object constant) {
+      this.constant = constant;
+    }
+
+    @Override
+    public boolean isNullAt(int i) {
+      return constant == null;
+    }
+
+    @Override
+    public int getInt(int i) {
+      return (int) constant;
+    }
+  }
+
+  private static class ConstantLongColumnVongector implements LongColumnVector {
+
+    private final Object constant;
+
+    private ConstantLongColumnVongector(Object constant) {
+      this.constant = constant;
+    }
+
+    @Override
+    public long getLong(int i) {
+      return (long) constant;
+    }
+
+    @Override
+    public boolean isNullAt(int i) {
+      return constant == null;
+    }
```

Comment:
Nit: Could we keep all those ConstantXXColumnVector has the same order for `getLong` method and `isNullAt` method ?  I see `ConstantIntColumnVector` put the `isNullAt` ahead of `getInt`, but `ConstantLongColumnVongector` is the reversed order.